libs/SalesforceSDKCore/SalesforceSDKCore/Classes/RestAPI/SFRestRequest.h

```diff
@@ -104,6 +104,15 @@ extern NSString * const kSFDefaultRestEndpoint;
 @property (nullable, nonatomic, strong, readwrite) NSURLSessionDataTask *sessionDataTask;
 
 /**
+ * The base URL of the request
+ * This should be used only in situations where the user's API URL is not the desired endpoint
```

Comment:
I'd probably amend this to not be quite so "caveat emptor"-driven.  This should work fine for people when they know what it does, as long as they use it properly.

First, can we amend the statement above to be more descriptive?  E.g.:
```
The base URL of the request, to be prepended to the value of the `path` property.  By default, this
will be the API URL associated with the current user's account.
```
Next, I'd probably only leave the "Specifically" bit below, reworded like:
```
One use would be when in a community setting and you want to send a request against the base API URL.
```
Thoughts?